recipes/notify.rb

```diff
@@ -137,12 +137,22 @@
       if ENV['USE_S3']
         s3 = Aws::S3::Client.new(region: 'eu-west-1')
 
+        promote_to = case ENV['ORGANISATION']
+                     when "integration" then "staging"
+                     when "staging" then "production"
+                     else "integration"
+                     end
+
         unless ENV['TAG'] == "deployed-to-#{ENV['ORGANISATION']}"
-          puts "Copying object to deployed-to-#{ENV['ORGANISATION']} branch"
-          s3.copy_object({ :bucket      => ENV['S3_ARTEFACT_BUCKET'],
+          puts "Copying object to deployed-to-#{promote_to} branch"
+          s3.copy_object({ :bucket      => "govuk-#{promote_to}-artefact",
+                           :key         => "#{application}/#{ENV['TAG']}/#{application}",
+                           :copy_source => "#{ENV['S3_ARTEFACT_BUCKET']}/#{application}/deployed-to-#{ENV['ORGANISATION']}/#{application}" })
+
+          puts "Copying object to  release #{ENV['TAG']}-#{ENV['ORGANISATION']} branch"
```

Comment:
I think it would be clearer to say something like:

"Promoting #{ENV['TAG']} to #{promote_to}: govuk-#{promote_to}-artefact"

This will let the user know exactly why the operation is being performed.